fla/modules/conv/short_conv.py

```
@@ -0,0 +1,241 @@
+            assert activation in ['silu', 'swish'], f"Activation `{activation}` not supported yet."
+            self.activation = activation
+
+        if 'use_fast_conv1d' in kwargs:
+            warnings.warn(
+                "The `use_fast_conv1d` parameter is deprecated and will be ignored. "
+                "Please use the `backend` parameter instead.",
+            )
+        import os
+        self.backend = os.environ.get('FLA_CONV_BACKEND', backend)
+        if backend not in ['cuda', 'triton']:
+            raise ValueError(f"Invalid backend: {backend}, must be one of ['cuda', 'triton']")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)